We want the generated Makefile to contain "\n", not an actual newline,
so an extra level of escaping is needed.
--- setup/BLDUTIL3.i.orig	2018-07-17 14:06:32.000000000 -0500
+++ setup/BLDUTIL3.i	2018-07-17 14:24:43.000000000 -0500
@@ -1030,7 +1030,7 @@
 			WriteCStrToDestFile("printf \"");
 			ptext();
 			if (newline) {
-				WriteCStrToDestFile("\\n");
+				WriteCStrToDestFile("\\\\n");
 			}
 			WriteCStrToDestFile("\" >");
 			WritePathArgInMakeCmnd(pto);
